server/election/leadership.go

```diff
@@ -221,5 +234,4 @@ func (ls *Leadership) Reset() {
 		ls.keepAliceCancelFunc()
 	}
 	ls.getLease().Close()
-	ls.setLease(nil)
```

Comment:
Involved in https://github.com/tikv/pd/pull/4044/files#diff-1df57c033011699202ef9b7617f0cee905ed0e86445362af5ec416d2aa403b84R224.